### DISCUSSION TITLE
Should LiveSampleURL macro be removed?

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#270] [Platform] [answered]
Should the `LiveSampleURL` macro be deprecated? It has no remaining occurrences in en-US or translated-content (to my knowldge). ![rg -il "\{\s*LiveSampleURL"
](https://user-images.githubusercontent.com/66077254/197890039-69dd5419-b6df-4991-bcb6-c1fe45bb45d8.png)
![image](https://user-images.githubusercontent.com/66077254/197890817-95c0ea6e-4520-4806-9769-9f7e85520832.png)

**Comment by yin1999** [comment, marked as answer]
Used by [`{{EmbedLiveSample}}`](https://github.com/mdn/yari/blob/b7056d69a659fa0e2b91f69dab4496b37770028c/kumascript/macros/EmbedLiveSample.ejs#L71) and `{{LiveSampleLink}}` macros

**Reply by MrBrain295** [reply]
Oh
